invokeai/app/services/images.py

```diff
@@ -397,3 +404,28 @@ def delete_images_on_board(self, board_id: str):
         except Exception as e:
             self._services.logger.error("Problem deleting image records and files")
             raise e
+        
+    def delete_many(self, is_intermediate: bool):
```

Comment:
I feel like I'm gonna call this with is_intermediate=False on accident one day haha. Would that delete 100 generated images?

Reply:
Yep it would (and soon we will have bulk actions that will allow that from the front-end)